client/tailwind.config.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
         100: '#D1FAE5',
         50: '#EBF6F1',
       },
+      transparent: '#00000000',
```

Comment:
what the hell is this?

Reply:
I added it at some time to avoid issues with borders. Going from no border to some border on focus resized the element. I'm not sure if I used it tho

Reply:
let's try to remove it, at least it should have a length of 6

Reply:
Those extra numbers are the alpha value, the idea is for it to be invisible, not black or white

Reply:
I came upon some.uses of -transparent before adding that, and I remember that without it explicit in the config twin.macro doesn't detect it and throws an error

Reply:
I came upon some uses of -transparent before adding that, and I remember that without it explicit in the config twin.macro doesn't detect it and throws an error